### PR TITLE
update docs for integration of headless mode

### PR DESCRIPTION
**Pull request by hades-6**

### CODE REVIEW

File: docs/sdk/browser/android/index.md [hunk 1]
```diff
@@ -242,6 +232,72 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
...
+
+profileData.put("ContactDetails.CellPhones.CellPhone.Number", "0416525889");
+profileData.put("ContactDetails.Emails.Email.Address", "test@test.com");
+
+profileData.put("AddressDetails.HomeAddress.AddressLine1", "245 Chapel Street");
+profileData.put("AddressDetails.HomeAddress.Suburb", "Prahran");
+profileData.put("AddressDetails.HomeAddress.PostalCode", "3181");
+profileData.put("AddressDetails.HomeAddress.AdministrativeArea", "Glen Iris");
+
+Fillr.getInstance().performAutofillOnWebView(webview, mapping);
```
Comment (alexzhaobin) [inline docs/sdk/browser/android/index.md]: profileData is not passed in?

File: docs/sdk/browser/android/index.md [hunk 2]
```diff
@@ -242,6 +232,72 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
...
+          //Using the Fillr namespace; profile data values can be set. Such as,
+          //"PersonalDetails.Honorific", "PersonalDetails.FirstName" which is avialable through the FillrMapping object. 
+       }
+    });
+```
+
+### Inserting data into the webview
+
+In order to parse data into the webview; it we can utilize the method described below,
+
```
Comment (alexzhaobin) [inline docs/sdk/browser/android/index.md]: Can we move this code block inside: public void onFormDetected(FillrWebView webview, FillrMapping mapping) method above, there could be integrator not be able to figure this out themselves.

File: docs/sdk/browser/android/index.md (hunk 1 above)
Comment (hades-6) [reply docs/sdk/browser/android/index.md]: Mapping is an object of type FillrMapping.
Form namespace and the profile data values as a key value pairs respectively.
Can be set setProfileData. I will update the read me to reflect this cause it doesn't seem clear.

**Comment by hades-6** [comment]
ok moved it.

**Pull request closed without merging**